payments/credit-card/index.md

```diff
@@ -34,7 +34,7 @@ and global way to initate a transaction with a customer. Choose between our
 **Seamless View**, **Redirect**, or **Direct** integration options." %}
 
 > **Seamless View** is our solution for a payment experience that is integrated
```

Comment:
Please don't use `>` as they look bland and are meant for quotes. Use the alert include instead.